### DISCUSSION TITLE
可以通过直接向2.版本的unity项目中导入3.包来实现版本升级吗？

### DISCUSSION THREAD

**Discussion by ConanGentleman** [XCharts-Team/XCharts#296] [General]
如题

**Comment by monitor1394** [comment]
3.x版本相比2.x版本，改动比较大，不完全兼容2.x版本。
升级时，除了部分代码需要调整外，部分图表的参数可能需要重新设置。
另外，升级时，先备份好旧项目，然后删除旧的2.x版本，再导入3.x版本。
花一点时间还是可以升级的。
